testsuite/openshift/objects/dnspolicy.py

```diff
@@ -13,15 +40,25 @@ def create_instance(
         openshift: OpenShiftClient,
         name: str,
         parent: Referencable,
+        healthCheck: HealthCheck = None,
         labels: dict[str, str] = None,
-    ):
+    ):  # pylint: disable=invalid-name
         """Creates new instance of DNSPolicy"""
 
-        model = {
+        model: dict = {
             "apiVersion": "kuadrant.io/v1alpha1",
             "kind": "DNSPolicy",
             "metadata": {"name": name, "labels": labels},
             "spec": {"targetRef": parent.reference},
         }
 
+        if healthCheck:
+            model["spec"]["healthCheck"] = asdict(healthCheck)
```

Comment:
I was initially skeptical about this change because of how we treat other similar things in the test suite. But in the end it was definitely a change for the better. 

Perhaps we should consider refactoring more "additional configurations" in the same manner.